aiohttp/cookiejar.py

```diff
@@ -256,8 +256,27 @@ def filter_cookies(
             and request_origin not in self._treat_as_secure_origin
         )
 
+        # d: domain
+        # p: path
+        d = hostname
+        p = request_url.path
+
+        cookies = list(self._cookies[(d, p)].values())
+
+        # shared cookie, it should have max of 1 entry
+        shared_cookie = self._cookies.get(("", "/"))
```

Comment:
Maybe I'm just forgetting. But, is there anything in the code, or something in the spec which mentions this case?